fetch-artifact.sh

```diff
@@ -1,9 +1,14 @@
 #!/usr/bin/env bash
 
 # Args:
-# $1: container registry url
+# $1: container url
 # $2: platform
 # $3: output path
 
+if [ $# -ne 3 ]; then
+	echo "$0 <container url> <platform> <output path>"
+	exit 1
+fi
+
```

Comment:
Since we're pushing artifacts using buildah/podman, can we also fetch the artifacts using podman? Or do users have to install oras just to fetch?

Reply:
I don't think either tool provides the facility. I'll ask around though

Reply:
Yup, from @nalind on #forum-containers: "that's not something that's implemented yet, but it's on the drawing board" with regards to Buildah's ability to pull artifacts.